OpenRA.Mods.RA/Effects/SatelliteLaunch.cs

```diff
@@ -43,7 +43,7 @@ public void Tick(World world)
 			if (++frame == 19)
 			{
 				var palette = info.SatellitePaletteIsPlayerPalette ? info.SatellitePalette + launcher.Owner.InternalName : info.SatellitePalette;
-				world.AddFrameEndTask(w => w.Add(new GpsSatellite(world, pos, info.SatelliteImage, info.SatelliteSequence, palette)));
+				world.AddFrameEndTask(w => w.Add(new GpsSatellite(world, pos, info.SatelliteImage, info.SatelliteSequence, palette, info.RevealDelay * 25, launcher.Owner)));
```

Comment:
Can we please change `info.RevealDelay` to be specified in ticks instead of "seconds measured at the default game speed"?  This isn't a merge blocker, but I'd really like to see it happen.